--- a/src/pulseaudio.c
+++ b/src/pulseaudio.c
@@ -82,14 +82,9 @@
 
         case PA_CONTEXT_READY:
         {
-            char* tooltip = context_info_str(context);
-            char* escaped = g_markup_escape_text(tooltip, -1);
-            char* markup = g_strdup_printf(
-                    "<span font_family=\"monospace\" font_size=\"x-small\">%s</span>", escaped);
-            systray_impl_set_tooltip(mis->systray, markup);
-            g_free(escaped);
+	    char* tooltip = g_strdup("Sound Control");
+            systray_impl_set_tooltip(mis->systray, tooltip);
             g_free(tooltip);
-            g_free(markup);
 
             pa_context_set_subscribe_callback(context, pulseaudio_event_cb, mis);
             pa_operation_unref(pa_context_subscribe(context,
##
## This diff does not apply. Update or drop..?
#--- a/src/systray.c
#+++ b/src/systray.c
#@@ -44,17 +44,9 @@
#     systray_rootmenu_add_submenu(mis, MENU_OUTPUT, "Recording Stre_ams", "player_record");
#     systray_menu_add_separator(mis->menu);
# 
#-    static const char* COMMAND_PAMAN = "paman";
#     static const char* COMMAND_PAVUCONTROL = "pavucontrol";
#-    static const char* COMMAND_PAVUMETER = "pavumeter";
#-    static const char* COMMAND_PAVUMETER_REC = "pavumeter --record";
#-    static const char* COMMAND_PAPREFS = "paprefs";
# 
#-    systray_menu_add_application(mis->menu, "_Manager...", NULL, COMMAND_PAMAN);
#     systray_menu_add_application(mis->menu, "Volume _Control...", NULL, COMMAND_PAVUCONTROL);
#-    systray_menu_add_application(mis->menu, "Volume Meter (_Playback)...", NULL, COMMAND_PAVUMETER);
#-    systray_menu_add_application(mis->menu, "Volume Meter (_Recording)...", NULL, COMMAND_PAVUMETER_REC);
#-    systray_menu_add_application(mis->menu, "Con_figure Local Sound Server...", NULL, COMMAND_PAPREFS);
# 
#     /* TODO: settings
#     systray_menu_add_separator(mis->menu);
